_includes/media/_video-player-card.html

```diff
@@ -1,3 +1,22 @@
+{% comment %}
+// This assures the messages content type uses the background image instead of the thumbnail image like all other content types.
+{% endcomment %}
+{% assign handle = item.content_type %}
+{% case handle %}
+  {% when 'message' %}
+    {% assign banner_image = item.background_image.url %}
+  {% when 'episode' %}
+    {% assign banner_image = item.podcast.image.url %}
+  {% else %}
+    {% assign banner_image = item.image.url %}
+{% endcase %}
+
+{% if include.thumbnail == true %}
+    <h1>I should show thumbnail</h1>
```

Comment:
Do you want this to go into prod? Sounds/feels like some debug code

Reply:
good catch, I removed it